tests/python/python_tests.bzl

```diff
@@ -149,6 +149,7 @@ def _test_default(env):
         "base_url",
         "ignore_root_user_error",
         "tool_versions",
+        "platforms",
```

Comment:
Ah, so this does not expose any extra APIs yet, so hence no extra tests here, LGTM.